dashboard/estar.html

```diff
@@ -0,0 +1,208 @@
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+
+	<!-- d3  -->
+	<script src="http://d3js.org/d3.v3.min.js" type="text/javascript"></script>
+	<script src="https://d3js.org/d3-queue.v2.min.js" type="text/javascript"></script>
+
+	<!-- jStat https://github.com/jstat/jstat -->
+	<!-- <script src="https://cdn.jsdelivr.net/jstat/latest/jstat.min.js"></script>
+	<script src="../lib/jstat.min.js"></script> -->
+
+	<!-- working on the airplane section... -->
+	<script>var offline = false;</script>
+<!--
+	<script src="offlinedata.js"></script>
+	<link rel="stylesheet" href="../lib/bootstrap.min.css">
+	<link rel= "stylesheet" type="text/css" href="../lib/font-awesome.min.css">
+	<script src="../lib/jquery.min.js"></script>
+	<script src="../lib/jquery-ui.js"></script>
+	<script src="../lib/bootstrap.min.js"></script>
+	<script src="../lib/d3.v3.min.js" type="text/javascript"></script>
+	<script src="../lib/d3-queue.v2.min.js" type="text/javascript"></script>
+ -->
+	<!-- end working on the airplane section... -->
+
+	<!-- todo: find the cdn for soda-js -->
+	<script src="../lib/soda-js.bundle.js" type="text/javascript"></script>
+
+	<link rel="stylesheet" href="css/dashboard.css">
+</head>
+<body>
+
+	<div class="container-fluid">
+		<div class="row">
+			<nav class="navbar navbar-default navbar-fixed-top">
+				<div class="container">
+					<div class="navbar-header">
+						<a class="navbar-brand" href=""><img src="../assets/sf_logo_white.png" alt="SF Dept of Environment"></a>
+					</div>
+				</div>
+			</nav>
+		</div>
+		<div id="view-welcome" style="margin-top: 1em;">
+			<p>This dashboard is a project of the <a href="http://datascience.codeforsanfrancisco.org">Data Science Working Group</a> at Code for San Francisco.</p>
+			<p>The intent is help building property managers understand the energy usage in compliance with the <a href="https://sfenvironment.org/energy/energy-efficiency/commercial-and-multifamily-properties/existing-commercial-buildings-energy-performance-ordinance">Existing Commercial Buildings Ordinance</a>.</p>
+			<p>To see an example of a specific property, <a href="index.html?apn=3705/039">click here</a>.</p>
+		</div>
+		<div id="view-load" class="hidden" style="margin-top: 1em;">
+			Loading...
+		</div>
+      <div id="view-content" class="row hidden">
+      	<div class="col-xs-3">
+			<div class="row">
+	            <div class="panel-body side flex-grow">
+                	<div class="col-xs-12">
+                		<div class="row center">
+                			<div class="profile-header">BENCHMARKING PROFILE</div>
+                		</div>
+                		<div class="row center">
+                			<i class="fa fa-building" aria-hidden="true"></i>
+                			<!-- <img src="../assets/hotel-icon.svg" alt="hotel-icon" id="building-icon"> -->
```

Comment:
is the font-awesome image the new path forward?  it's only changed on this page.